save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestSuitesController.kt

```diff
@@ -95,15 +95,6 @@ class TestSuitesController(
             )
         }
 
-    /**
-     * @param testSuiteDtos suites, which need to be marked as obsolete
-     * @return response entity
-     */
-    @PostMapping("/internal/markObsoleteTestSuites")
```

Comment:
now, not need to mark test suites as old, but we need to able to remove test suites by version